src/components/Modals/CreateItemModal/CreateItemModal.tsx

```diff
@@ -472,6 +477,7 @@ export default class CreateItemModal extends React.PureComponent<Props, State> {
     const { model, contents } = this.state
 
     const isCustom = !!contents && THUMBNAIL_PATH in contents
+    // TODO: Only executes if custom? why?
```

Comment:
Only executes if NOT custom. That means that if the users provides a custom thumbnail (either a `thumbnail.png` file in the zip file or uploading a custom one via de edit icon in the UI) we use that one instead of generatin a new one from the model.